src/main.rs

```diff
@@ -94,8 +94,10 @@ impl ExtrinsicOpts {
 
 #[derive(Clone, Debug, StructOpt)]
 pub struct VerbosityFlags {
+    /// No output printed to stdout
```

Comment:
This is not actually true, it just passes `--quiet` to the underlying `cargo` invocation. We would need to add conditionals around all our `println!`s for it to be quiet in this sense.

Reply:
Some(Verbosity::Quiet) => cmd.arg("--quiet"),
Some(Verbosity::Verbose) => cmd.arg("--verbose"),
I see. I just copy cargo --quiet.
What do you think it should be?

Reply:
Cargo No output printed to stdout.   How about this one?

Reply:
What about `No cargo output printed to stdout`.

Also opened an issue about making it truly quiet: https://github.com/paritytech/cargo-contract/issues/134